### PR TITLE
Initial build of d-spy

### PR DESCRIPTION
**Pull request by chergert**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.gnome.dspy.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            ]
+        },
+        {
+            "name" : "libadwaita",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+                    "branch" : "main",
```
Comment (hfiguiere) [inline org.gnome.dspy.json:67]:
```suggestion
```

File: org.gnome.dspy.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        },
+        {
+            "name" : "sassc",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/sassc.git",
+                    "branch" : "meson",
```
Comment (hfiguiere) [inline org.gnome.dspy.json:55]:
```suggestion
```

File: org.gnome.dspy.json [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    "modules": [
+        {
+            "name" : "libsass",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/libsass.git",
+                    "branch" : "meson",
```
Comment (hfiguiere) [inline org.gnome.dspy.json:42]:
```suggestion
```

File: org.gnome.dspy.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        "--socket=session-bus",
+        "--socket=system-bus"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
```
Comment (hfiguiere) [inline org.gnome.dspy.json:21]:
```suggestion
```

these are not necessary

File: org.gnome.dspy.json [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "d-spy",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=session-bus",
+        "--socket=system-bus"
```
Comment (hfiguiere) [inline org.gnome.dspy.json:13]: I'm gonna assume that given what dspy is this is essential :-)

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build org.gnome.dspy

**Comment by chergert** [comment]
bot, build org.gnome.dspy

File: org.gnome.dspy.json (hunk 5 above)
Comment (chergert) [reply org.gnome.dspy.json:13]: yup yup

**Pull request closed without merging**